lib/ansible/modules/storage/netapp/netapp_e_alerts.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/python
+
+# (c) 2018, NetApp, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: netapp_e_alerts
+short_description: manage E-Series email notification settings
+description:
+    - Certain E-Series systems have the capability to send email notifications on potentially critical events.
+    - This module will allow the owner of the system to specify email recipients for these messages.
+version_added: '2.7'
+author: Michael Price (@lmprice)
+extends_documentation_fragment:
+    - netapp.eseries
+options:
+    state:
+        description:
+            - Enable/disable the sending of email-based alerts.
+        default: enabled
+        required: false
+        choices:
+            - present
+            - absent
+            - enabled
+            - disabled
+    server:
+        description:
+            - A fully qualified domain name, IPv4 address, or IPv6 address of a mail server.
+            - To use a fully qualified domain name, you must configure a DNS server on both controllers using
+             M(netapp_e_mgmt_interface).
+             - Required when I(state=enabled/present).
```

Comment:
I think it's confusing having enabled and present

Reply:
The idea was to have consistency with all of the other places that state is utilized in Ansible modules. state=present/absent seems to be a common convention elsewhere (hence why prior developers used it in our volume module, for instance).

Reply:
Sure, though the issue is you are supporting both present and enabled. What's the difference?
Just pick one pair and use that

Reply:
enabled/disabled for this as we are talking about a service (email)